server/tournament.py

```diff
@@ -697,6 +689,79 @@ def points_perfs(self, game):
 
         return (wpoint, bpoint, wperf, bperf)
 
+    def points_perfs_janggi(self, game):
+        wplayer = self.players[game.wplayer]
+        bplayer = self.players[game.bplayer]
+
+        wpoint = (0, SCORE)
+        bpoint = (0, SCORE)
+        wperf = game.black_rating.rating_prov[0]
+        bperf = game.white_rating.rating_prov[0]
+
+        if game.result == "1-0":
+            wplayer.nb_win += 1
+
+            if self.system == ARENA:
+                if wplayer.win_streak == 2:
+                    if game.status == VARIANTEND:
+                        bpoint = (2, SCORE)
+                        wpoint = (4 * 2, SCORE)
+                        wplayer.win_streak = 0
```

Comment:
You'll need to cover the case where the loser is on a win streak too.

Reply:
Thx, fixed it.